locale/update_translations.sh

```diff
@@ -57,10 +58,9 @@ done
 
 cd ${srcdir}
 
-echo "Translations compiled successfully. Now some manual steps:"
-echo " 1. less BOINC-Manager-pofilter.txt # check output from pofilter and adjust translations then start this script again"
-echo " 2. git add -u # only update already tracked files (add new files when needed too)"
-echo " 3. git commit -m \"Locale: Update localization files [skip ci]\""
-echo " 4. git push"
+git add .
```

Comment:
After it git add, what happen? The code is not updated. Why you delete git push?